### DISCUSSION TITLE
SetSchema 질문

### DISCUSSION THREAD

**Discussion by hanaaple** [naverz/zepeto-studio-kor#734] [5. Q&A]
index.d.ts에 SetSchmea가 존재하는 것을 발견했는데 인스펙터에는 Set이 보이지 않습니다. 이유를 알 수 있을까요?
 
![image](https://user-images.githubusercontent.com/48403325/167881832-fcbf99c2-9029-42c0-b431-eb0dc9c445a2.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Schema (Set, Collection) 타입의 클라이언트 구현이 필요한 이슈로 차후 지원할 예정입니다.
감사합니다.
